src/client/actions/User.ts

```diff
@@ -65,7 +65,7 @@ export default class User {
 
```

Comment:
If you could add a TODO here to indicate that for the next major release we should rename the arg season to seasons

Reply:
The thing is that I don't know whether it is possible to specify two seasons? Whereas the `season` parameter name would be fine. I naively tried with commas only.

Reply:
From what I know it isn't possible.

Reply:
Then is the TODO addition necessary? Current method signature is clearer than the API one actually!

Reply:
I guess not.
I'll merge it